linode/manager
# Cloning a domain in Cloud Manager leads to Not Found page instead of new domain page

## Description

Upon initiating a domain clone (for a master domain), the user is redirected to the details page, which initially displays a 'Not Found' page before resolving to display the details page. This PR adds a timeout to the clone action to prevent the Not found display.

## Type of Change
- Bug fix ('fix')

If the any above types of change apply to this pull request, please ensure to include one of the listed keywords in the pull request title.

## Note to Reviewers

To test:
On develop, clone a master domain. You should see 'Not Found' before the details page displays (or you may have to go back to the landing and re-click the newly made clone to see the details page). On this branch, you should never see 'Not found' display, only a brief loading screen.
